packages/components/psammead-play-button/src/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { string, oneOf, func } from 'prop-types';
+import styled, { css } from 'styled-components';
+import { C_EBON, C_WHITE, C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { GEL_MINION } from '@bbc/gel-foundations/typography';
+import { getSansBold } from '@bbc/psammead-styles/font-styles';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const Button = styled.button`
+  background-color: ${C_EBON};
+  border: none;
+  color: ${C_WHITE};
+  cursor: pointer;
+  display: block;
+  ${({ service }) => getSansBold(service)}
+  ${GEL_MINION};
+  height: 80px;
+  padding: 0;
+  transition: background-color 300ms;
+  width: 80px;
+
+  &:hover,
+  &:focus {
+    background-color: ${C_POSTBOX};
+    transition: background-color 300ms;
```

Comment:
Can `300` be defined as a constant at the top of this file please :)

Reply:
Done. 👍  I kept it a string, so that it needn't be coerced and can keep it's unit, but let me know if you'd rather it were a number.